src/serverMain/resources/userFrontend/liveCheckInsView.js

```diff
@@ -0,0 +1,51 @@
+let fullLocationId = new URLSearchParams(window.location.search).get("l")
```

Comment:
whole file: Pls do a `;` at the end of each line, omitting this can have strange side effects...